html/contact.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Contact me</title>
+    <html lang="en"></html>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" type="text/css" media="screen" href="../styles/normalize.css" />
+    <link rel="stylesheet" type="text/css" media="screen" href="../styles/style.css" />
+    <script src="main.js"></script>
+</head>
+<body class="container">
+        <header class="left-navigation header nav-down">
+                <div class="left-navigation-wraper">
+                    <nav>
```

Comment:
Please make sure your editor is configured to use two-space tabs. That will make reading all this HTML a little easier.